### ISSUE TITLE
exclusiveMaximum/Minimum differnece with JSON Schema Validation

### ISSUE THREAD

**Issue by AleksandrPB** [zircote/swagger-php#966]
According to http://json-schema.org/draft/2020-12/json-schema-validation.html#rfc.section.6.2.3 type of exclusiveMaximum/exclusiveMinimum field must be numeric and represent exclusive maximum value not a boolean. Is there any reason to keep it as a boolean type?

```php
/**
     * See http://json-schema.org/latest/json-schema-validation.html#anchor17.
     *
     * @var bool
     */
    public $exclusiveMaximum = Generator::UNDEFINED;
```

**Comment by DerManoMann** [comment]
Probably not! Thanks for noticing!

**Comment by DerManoMann** [comment]
Fixed via #967
